lib/scrabble.rb

```diff
@@ -1,7 +1,18 @@
+require 'pry'
+
 class Scrabble
 
   def score(word)
-    1
+    if word == nil
+      0
+    else
+      letters = word.chars
+      value = []
+      letters.map do |letter|
```

Comment:
Map returns an array, you dont need to have it shovel things into an external array